scripts/release/build/parse-build-parameters.js

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env node
+
+'use strict';
+
+const chalk = require('chalk');
+const commandLineArgs = require('command-line-args');
+const commandLineUsage = require('command-line-usage');
+const figlet = require('figlet');
+const {paramDefinitions} = require('../config');
+
+module.exports = () => {
+  const params = commandLineArgs(paramDefinitions);
+
+  if (!params.version) {
+    const usage = commandLineUsage([
+      {
+        content: chalk
+          .hex('#61dafb')
+          .bold(figlet.textSync('react', {font: 'Graffiti'})),
```

Comment:
so _that’s_ how you’re generating the fun artwork!

Reply:
I actually generated it manually 😅  before remembering the `figlet` module heh